components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -1587,8 +1587,8 @@ void AdsServiceImpl::GetSiteHistory(int max_count,
             }
 
             base::ranges::sort(site_history);
-            site_history.erase(base::ranges::unique(site_history),
-                               site_history.cend());
+            auto to_remove = std::ranges::unique(site_history);
+            site_history.erase(to_remove.begin(), site_history.end());
```

Comment:
nit: despite `site_history.end()` being equal to `to_remove.end()` doing `erase(to_remove.begin(), to_remove.end())` looks a bit more natural

Reply:
Done. (Got bitten by auto-complete there)